### DISCUSSION TITLE
Using Docker to build the production

### DISCUSSION THREAD

**Discussion by Tomut0** [Tomut0/SimpleClansStats#19] [Announcements]
It's a good practice to use "containers" instead of installation scripts.
I need to think about it: maybe it's unnecessary (too complicated for users?).
<hr>
Useful links:

- https://docker-curriculum.com/ (en)
- https://habr.com/ru/post/309556/ (only ru)
- https://habr.com/ru/post/310460/ (only ru)
- https://habr.com/ru/post/461687/ (only ru)
- https://docs.github.com/en/packages/guides/configuring-docker-for-use-with-github-packages
- https://laravel.io/articles/create-a-project-in-laravel-8x-with-laravel-sail-docker
- https://hub.docker.com/
- https://medium.com/@shakyShane/laravel-docker-part-2-preparing-for-production-9c6a024e9797

**Comment by gotogrub** [comment]
Containers for jerks
